lib/Entity/DataSet.php

```diff
@@ -432,6 +436,9 @@ public function getData($filterBy = [], $options = [])
         if ($filter != '') {
             // Support display filtering.
             $filter = str_replace('[DisplayId]', $displayId, $filter);
+            // Support display filtering by [DisplayName]
+            $filter = str_replace('[DisplayName]', $displayName , $filter);
```

Comment:
We will need to make sure we add single quotes around our display name (if they aren't already in the filter)